build.ps1

```diff
@@ -3,21 +3,25 @@
 [CmdletBinding()]
 param (
   [Parameter()]
-  [ValidateSet('build', 'package')]
+  [ValidateSet('build', 'quick', 'package')]
   [string]
   $Target = 'build'
 )
 $Env:WORKINGDIR = $PSScriptRoot
 
+$arch = go env GOHOSTARCH
+$platform = go env GOHOSTOS
+$binPath = Join-Path $PSScriptRoot "dist" "pct_${platform}_${arch}"
+
 switch ($Target) {
   'build' {
-    $arch = go env GOHOSTARCH
-    $platform = go env GOHOSTOS
-    $binPath = Join-Path $PSScriptRoot "dist" "pct_${platform}_${arch}"
     # Set goreleaser to build for current platform only
     goreleaser build --snapshot --rm-dist --single-target
     git clone -b main --depth 1 --single-branch https://github.com/puppetlabs/baker-round (Join-Path $binPath "templates")
   }
+  'quick' {
+    go build -o ${binPath}/pct
```

Comment:
```suggestion
    If ($Env:OS -match '^Windows') {
      go build -o "$binPath/pct.exe"
    } else {
      go build -o "$binPath/pct"
    }
```

As currently implemented, a Windows machine doesn't know what to do with `pct` and prompts for the right file to open it; with `pct.exe`, it executes as expected.